rasa/utils/tensorflow/tf_layers.py

```diff
@@ -57,7 +56,27 @@ def call(self, inputs: tf.SparseTensor) -> tf.Tensor:
         return outputs
 
 
-class ReluFfn(tf.keras.layers.Layer):
+class DenseWithSparseWeights(tf.keras.layers.Dense):
+    def __init__(self, sparsity: int = 0.8, **kwargs) -> None:
+        super().__init__(**kwargs)
+        self.sparsity = sparsity
+
+    def build(self, input_shape: tf.TensorShape) -> None:
+        super().build(input_shape)
+        # create random mask to set weights to 0
```

Comment:
```suggestion
        # create random mask to set some weights to 0
```
might be a bit clearer